Library/Homebrew/formula_installer.rb

```diff
@@ -201,12 +202,12 @@ def check_install_sanity
   end
 
   def build_bottle_preinstall
-    @etc_var_glob ||= "#{HOMEBREW_PREFIX}/{etc,var}/**/*"
-    @etc_var_preinstall = Dir[@etc_var_glob]
+    @etc_var_dirs ||= ["#{HOMEBREW_PREFIX}/etc", "#{HOMEBREW_PREFIX}/var"]
+    @etc_var_preinstall = Find.find(*@etc_var_dirs.select { |d| File.directory?(d) }).to_a
   end
 
   def build_bottle_postinstall
-    @etc_var_postinstall = Dir[@etc_var_glob]
+    @etc_var_postinstall = Find.find(*@etc_var_dirs.select { |d| File.directory?(d) }).to_a
```

Comment:
```suggestion
    @etc_var_postinstall = Find.find(*@etc_var_dirs.select(&:directory?)).to_a
```

Reply:
Thanks, I'm not very fluent in Ruby! I'll make that change.